plugins/net_plugin/net_plugin.cpp

```diff
@@ -3287,29 +3290,33 @@ namespace eosio {
    // call only from main application thread
    void net_plugin_impl::update_chain_info() {
       controller& cc = chain_plug->chain();
-      uint32_t lib_num = 0, head_num = 0;
+      uint32_t lib_num = 0, head_num = 0, fork_head_num = 0;
       {
          fc::lock_guard g( chain_info_mtx );
          chain_info.lib_id = cc.last_irreversible_block_id();
          chain_info.lib_num = lib_num = block_header::num_from_id(chain_info.lib_id);
-         chain_info.head_id = cc.fork_db_head_block_id();
+         chain_info.head_id = cc.head_block_id();
          chain_info.head_num = head_num = block_header::num_from_id(chain_info.head_id);
+         chain_info.fork_head_id = cc.fork_db_head_block_id();
+         chain_info.fork_head_num = fork_head_num = block_header::num_from_id(chain_info.fork_head_id);
       }
-      fc_dlog( logger, "updating chain info lib ${lib}, head ${h}", ("lib", lib_num)("h", head_num) );
+      fc_dlog( logger, "updating chain info lib ${lib}, head ${h} fdb_head ${f}", ("lib", lib_num)("h", head_num)("f", fork_head_num) );
```

Comment:
Maybe we should be consistent in log messages and use `fdb_head` when we print the fork_db head?